sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -17,10 +17,6 @@ import { EventPosition } from "./eventPosition";
  * Options to pass when creating an iterator to iterate over events
  */
 export interface EventIteratorOptions {
-  /**
-   * Number of events to fetch at a time in the background
-   */
-  prefetchCount?: number;
```

Comment:
Can we comment this out instead of deleting? It will serve as a better glaring reminder that we need to get this done in the future :)

Reply:
Done!